src/components/services/Services.tsx

```diff
@@ -127,14 +132,34 @@ export function Services(props: Props) {
     'Services 2'
   )
 
+  const closePowerModal = () => {
+    powerAlertRef.current?.resolve()
+    powerAlertRef.current = undefined
+  }
+
```

Comment:
I don't like the idea of closing the modal from the outside. I would much rather do the `.resolve()` from within the body of `Airship.show`. You could move the `onPress` inside the `Airship.show` pretty easily, but the other two cases of `closePowerModal()` are harder to get rid of. Why is this necessary? If it truly is necessary, maybe we could find another way to auto-close the modal, besides leaking the whole bridge out.